.github/workflows/ci.yaml

```diff
@@ -91,14 +91,10 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: newrelic/helm-charts
-          ref: invidian/nri-kubernetes-operator
           path: helm-charts-newrelic
 
-      - name: Install Helm
```

Comment:
Where does helm binary comes from now? 🤔

Reply:
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md

Reply:
there is a series of software preinstalled in the runners

Reply:
TIL, thanks!